5-logging/app.js

```diff
@@ -50,12 +50,19 @@ const sessionConfig = {
   signed: true
 };
 
-// In production use the App Engine Memcache instance to store session data,
+// In production use the Memcache instance to store session data,
 // otherwise fallback to the default MemoryStore in development.
 if (config.get('NODE_ENV') === 'production' && config.get('MEMCACHE_URL')) {
-  sessionConfig.store = new MemcachedStore({
-    hosts: [config.get('MEMCACHE_URL')]
-  });
+  if (config.get('MEMCACHE_USERNAME') && (config.get('MEMCACHE_PASSWORD'))) {
```

Comment:
This really isn't available yet -- we shouldn't cross over to other people's products.